e2e/framework/manager/rook_test_infra.go

```diff
@@ -364,6 +376,20 @@ func (r *rookTestInfraManager) InstallRook(tag string) (err error, client contra
 	if err != nil {
 		panic(err)
 	}
+	err = r.copyImageToNode(r.getContainerIdByName(k8s_master_container_name), rookClientTag)
```

Comment:
add this to refactor ticket: https://github.com/rook/rook/issues/626.
Will clean it up then